lib/cli.js

```diff
@@ -72,7 +72,21 @@ exports.run = () => {
 			'update-snapshots',
 			'color'
 		],
-		default: Object.assign({color: true}, conf),
+		default: {
+			init: conf.init,
+			failFast: conf.failFast,
+			serial: conf.serial,
+			cache: conf.cache,
+			powerAssert: conf.powerAssert,
+			tap: conf.tap,
+			verbose: conf.verbose,
+			watch: conf.watch,
+			match: conf.match,
+			timeout: conf.timeout,
+			concurrency: conf.concurrency,
+			updateSnapshots: conf.updateSnapshots,
+			color: true || conf.color
```

Comment:
No, that'll still force the value to be true if `conf.color` is `false`.

If `color` is in `conf`, assume it's a usable value, else use `true`.